### PR TITLE
ES6: add Gustav to the platform team

### PR DESCRIPTION
**Pull request by gu-stav**
## What does this change?

Adds another contributor the the platform ES6 conversion team.

## What is the value of this and can you measure success?

Faster conversions.

### CODE REVIEW

**Comment by SiAdcock** [comment]
Should probably also remove yourself from `misc`!

**Review by SiAdcock — APPROVED** [review]

**Review by SiAdcock — APPROVED** [review]

**Review by SiAdcock — CHANGES_REQUESTED** [review]
Need to regenerate the JSON

**Comment by SiAdcock** [comment]
I have jumped on the back of this change and corrected Gareth and Jon's Git names, and regenerated the JSON.

**Review by SiAdcock — APPROVED** [review]

**Review by gtrufitt — CHANGES_REQUESTED** [review]
Other than doubling up on @gustavpursche !

**Review by gtrufitt — APPROVED** [review]

File: tools/es5to6-files-per-user.js [hunk 1]
```diff
@@ -31,11 +31,12 @@ const commercialFilesPerHuman = filesPerHuman(commercialModules, commercial);
 
 const platform = [
     'sndrs',
-    'gtrufitt',
+    'Gareth Trufitt',
     'Simon Adcock',
     'NataliaLKB',
     'GHaberis',
     'stephanfowler',
+    'Gustav Pursche',
```
Comment (gustavpursche) [inline tools/es5to6-files-per-user.js]: @gtrufitt On purpose - I can do a bit of extra work :)

**Pull request merged**